### ISSUE TITLE
Disable all exept a few select particals

### ISSUE THREAD

**Issue by DriftingSands** [SkinSpotlights/CreatorSuite-ReplayAPI#17]
Is there a way to disable all but a few particals? I have just been removing all of them in the entire list one by one, because I cant use ctrl+a or ctrl+shift to select and disable them all. 
It would save me alot of time when I want to disable everything exept a champions particals.

**Comment by SkinSpotlights** [comment]
Left Click Hold + Drag is the fastest way.

**Comment by DriftingSands** [comment]
yeah but it skips a few like that

**Comment by SkinSpotlights** [comment]
Yeah its why I marked it as enhancement so I can add more features such as shift clicking to it, just mentioned the only way to speed up the process because you were clicking them one by one.
